README.md

```diff
@@ -1,9 +1,7 @@
 # CSH Constitution
 The master branch's HEAD is the current legal constitution of CSH.
 
-Compiled version of both documents can be found below:
-#### [Articles](https://constitution.csh.rit.edu/articles.pdf)
-#### [By-Laws](https://constitution.csh.rit.edu/bylaws.pdf)
+A compiled version of the document can be found [here](constitution.csh.rit.edu).
```

Comment:
```suggestion
A compiled version of the document can be found [here](https://constitution.csh.rit.edu).
```
If the href is just `constitution.csh.rit.edu`, it gets interpreted as a local link (you can test it if you view the readme on your fork).

Reply:
Fixed it